extra/utils/hasPermission.js

```diff
@@ -9,16 +9,14 @@ export default function hasPermission(module_name, role, permissionType) {
   //let {getUsers : { all : [ x ] , read : [ b , c ] , write : [ d ] , delete : [] } } = permissions;
   if (module_name == "getUsers") {
```

Comment:
Please correct the logic. We do not need to check for specific module as string. We need to check for passed module name in function argument.